_posts/2018-04-30-happy-birthday-mr-johnson.md

```diff
@@ -7,38 +7,42 @@ categories: development testing
 
 ```java
 int clientAge = DateUtil.countAgeUsingPersonalCode("280752-7918")
 ```
 
-The insurance was more expensive than the test expected, right? The insurances usually become more expensive when the clients get older. Wait a second...
+The insurance policy was more expensive than the test expected, right? Insurance usually becomes more expensive as the clients get older. Wait a second...
 
-Today is July, 29th. Mr. Johnson (that was the name of the fictional client from the unit tests) celebrated his sixtieth birthday yesterday. And sixty years is one of the frontiers when the price for the insurance gets higher!
+Today is July 29th. Mr. Johnson (that was the name of the fictional client from the unit tests) celebrated his sixtieth birthday yesterday. And sixty years is one of the frontiers when the price for the insurance gets higher!
 
-Now everything was completely clear to me. The code didn't change indeed. And the unit tests didn't consume any services from the outer world: the only outer service was properly mocked. However, one external thing the tests still "consumed" was the time. Yesterday, and all the time before, the tests were passing because Mr. Johnson was still "young" and was eligible for cheaper insurance. Starting from today, however, he turned sixty, the price for him got higher, while nobody adjusted the tests and they still expected the price to stay the same...
+Now everything was completely clear to me. The code didn't change, and the unit tests didn't consume any services from the outer world; the only outer service was properly mocked. However, one external thing the tests still "consumed" was the time. Yesterday, and all the time before, the tests were passing because Mr. Johnson was still "young" and was eligible for cheaper insurance. Starting from today, however, he turned sixty, the price for his policy increased, while nobody adjusted the tests and they still expected the price to stay the same...
 
-That deployment took a little longer than expected. Still it was successfully finished. I also created a ticket to make the `DateUtil` independent on current moment of time and re-write the unit tests. And, the most important, I have learned a good lesson.
+That deployment took a little longer than expected. Still, it was successfully finished. I also created a ticket to make the `DateUtil` independent on current moment of time and re-write the unit tests. And, most importantly, I learned a good lesson.
 
 ### The Lesson Learned
 
-Never allow your unit tests depend on _anything_ external. Time is also the external thing, though it may seem stable and predictable.
+Never allow your unit tests to depend on _anything_ external. Time is also an external value, though it may seem stable and predictable.
```

Comment:
Can this paragraph be started with something like "To not allow any Mr. Johnsons spoil your evening again, never allow your unit tests..."? What would be the correct way to say that? Is "allow" 2 times acceptable here?

Reply:
I would suggest something like, "To make sure that no "Mr. Johnsons" spoil your evening again......" I think that works. What are your thoughts?